ci/test_cudf_polars_with_rapidsmpf.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+source rapids-init-pip
+
+rapids-logger "Download wheels"
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen "${RAPIDS_CUDA_VERSION}")"
+CUDF_POLARS_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="cudf_polars_${RAPIDS_PY_CUDA_SUFFIX}" RAPIDS_PY_WHEEL_PURE="1" rapids-download-wheels-from-github python)
+
+# Download libcudf and pylibcudf built in the previous step
+LIBCUDF_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="libcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-github cpp)
+PYLIBCUDF_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="pylibcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-github python)
+
+rapids-logger "Installing cudf_polars and its dependencies"
+
+# generate constraints (possibly pinning to oldest support versions of dependencies)
+rapids-generate-pip-constraints py_test_cudf_polars ./constraints.txt
+
+# Install rapidsmpf from nightly index
+rapids-logger "Installing rapidsmpf from nightly"
+rapids-pip-retry install \
+    -v \
+    --extra-index-url=https://pypi.nvidia.com \
+    --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple \
+    "rapidsmpf-${RAPIDS_PY_CUDA_SUFFIX}>=25.12.0a0,<25.13" \
+    "librapidsmpf-${RAPIDS_PY_CUDA_SUFFIX}>=25.12.0a0,<25.13"
+
+# Install cudf_polars with test and experimental extras
+rapids-pip-retry install \
+    -v \
+    --constraint ./constraints.txt \
+    --constraint "${PIP_CONSTRAINT}" \
+    "$(echo "${CUDF_POLARS_WHEELHOUSE}"/cudf_polars_"${RAPIDS_PY_CUDA_SUFFIX}"*.whl)[test,experimental]" \
+    "$(echo "${LIBCUDF_WHEELHOUSE}"/libcudf_"${RAPIDS_PY_CUDA_SUFFIX}"*.whl)" \
+    "$(echo "${PYLIBCUDF_WHEELHOUSE}"/pylibcudf_"${RAPIDS_PY_CUDA_SUFFIX}"*.whl)"
```

Comment:
I think instead of this what we want is a new `test_python_cudf_polars_rapidsmpf` similar to the existing [`test_python_cudf_polars`](https://github.com/rapidsai/cudf/blob/9179396068feb5d2c226e297389c5f22d8d186af/dependencies.yaml#L903-L908) section in the dependencies file and that will take care of installing `rapidsmpf`, you shouldn't need to pip/conda install anything in scripts.